### PR TITLE
Add click event on general search too

### PR DESCRIPTION
**Pull request by mapeveri**
Add click event on search icon in the general search like we did for #225

### CODE REVIEW

File: nodeshot/ui/default/static/ui/nodeshot/js/views/search.js [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
         events:{
             'keyup input': 'keyupOnSearch',
+            'click #general-search-input-button': 'ClickOnSearch',
```
Comment (nemesisdesign) [inline nodeshot/ui/default/static/ui/nodeshot/js/views/search.js]: `click @ui.icon` should work, see the `ui`section to understand: https://github.com/mapeveri/nodeshot/blob/master/nodeshot/ui/default/static/ui/nodeshot/js/views/search.js#L24

`ClickOnSearch`should be `clickOnSearch`(lowercase first letter)

File: nodeshot/ui/default/static/ui/nodeshot/js/views/search.js [hunk 2]
```diff
@@ -132,6 +133,12 @@
             }
         },
 
+        ClickOnSearch: function(e){
+            var valueSearch = $("#general-search-input").val();
+            this.typing = false;
+            this.search(valueSearch);
```
Comment (nemesisdesign) [inline nodeshot/ui/default/static/ui/nodeshot/js/views/search.js]: `this.search(this.ui.input.val());` should do.

File: nodeshot/ui/default/templates/index.html [hunk 3]
```diff
@@ -29,7 +29,7 @@
 
     <script id="search-template" type="text/template">
         <input type="text" id="general-search-input" placeholder="{% trans 'Search for data' %}">
-        <span class="icon-search">&nbsp;</span>
+        <span class="icon-search cursor-pointer" id="general-search-input-button">&nbsp;</span>
```
Comment (nemesisdesign) [inline nodeshot/ui/default/templates/index.html]: if the previous suggestions work, `id="general-search-input-button"`can be removed

**Comment by nemesisdesign** [comment]
Commit message could be something like:

`Added click event for general search, related to #225`

**Comment by mapeveri** [comment]
@nemesisdesign ready :)

**Comment by nemesisdesign** [comment]
Thanks @mapeveri :+1:

**Pull request merged**